demo/src/html/index.html

```diff
@@ -2,23 +2,9 @@
 <html>
     <head>
         <meta charset="UTF-8" />
-        <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
Oh, was this meant to be removed?

Reply:
Oops - that was by mistake! The code has been re-added.